### PR TITLE
HEL-211 | Fix feedback, remove unused record stuff

### PR DESCRIPTION
**Pull request by tuovinensanttu**
Main purpose was to fix feedback, which stopped working after last PR. This was caused by redirecting user to now removed old "Record detail view". 

I moved send feedback functionality to JS side, with this I was able to stop page from refreshing after form was submitted. Both `CollectionDetailView` and `SearchRecordDetailView` had their own functions for handling feedback, I combined these under `RecordFeedbackView` view. The new view returns `HttpResponse` which is used to show correct message after feedback is submitted.

### CODE REVIEW

File: hkm/static/hkm/js/hkm-v2.js [hunk 1]
```diff
@@ -339,6 +339,33 @@ palikka
...
+    event.preventDefault();
+
+    $.post('/record/feedback/', {
+      action: 'feedback',
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val(),
+      content: $('#id_feedback-form-content').val(),
+      full_name: $('#id_feedback-form-full_name').val(),
+      email: $('#id_feedback-form-email').val(),
+      hkm_id: $('input[name="hkm_id"]').val(),
+    }).done(function(response) {
```
Comment (klempine) [inline hkm/static/hkm/js/hkm-v2.js:353]: I think `done` is only triggered if there is no error. Should there also be a `fail` option here? Or maybe replace `done` with `always`?

**Review by klempine — APPROVED** [review]
I think this is fine, although that one jquery bit may need some finetuning (not sure though).

File: hkm/static/hkm/js/hkm-v2.js (hunk 1 above)
Comment (tuovinensanttu) [reply hkm/static/hkm/js/hkm-v2.js:353]: My jQuery is a little rusty. After looking into it, I think you are right. I added `.fail` to catch errors.

**Review by klempine — APPROVED** [review]

File: hkm/static/hkm/js/hkm-v2.js (hunk 1 above)
Comment (klempine) [reply hkm/static/hkm/js/hkm-v2.js:353]: Yup, looking good!

**Pull request merged**